code/datums/storage/subtypes/others/fish_case.dm

```diff
@@ -6,10 +6,10 @@
 /datum/storage/fish_case/can_insert(obj/item/to_insert, mob/user, messages, force)
 	. = ..()
 	if(!.)
-		return .
+		return
+
 	if(!HAS_TRAIT(to_insert, TRAIT_AQUARIUM_CONTENT))
-		if(messages && user)
-			user.balloon_alert(user, "can't hold!")
+		user.balloon_alert(user, "can't hold!")
```

Comment:
Doesn't this potentially runtime without the `user` check, and send a message when we don't want it to without the `messages` check?
Based on the parent proc I believe we want to run these checks before sending any message.